tests/Makefile

```diff
@@ -301,6 +301,7 @@ test-lz4c: lz4c datagen
 	./datagen -g256MB | $(LZ4)c -l -v    | $(LZ4)c   -t
 
 test-lz4c32: CFLAGS+=-m32
+test-lz4c32: MOREFLAGS+="-I/usr/include/x86_64-linux-gnu"
```

Comment:
The issue is a conflict of gcc-multilib and gcc-arm described here:
https://bugs.launchpad.net/ubuntu/+source/gcc-defaults/+bug/1300211

With Travis we have different containers but with Circle CI there is a single one.
The best solution I found 
https://bugs.launchpad.net/ubuntu/+source/gcc-defaults/+bug/1300211/comments/19
was to add `-I`.
Without it there is an error for 32-bit targets:
`/usr/include/linux/errno.h:1:23: fatal error: asm/errno.h: No such file or directory`